common/types.d.ts

```diff
@@ -182,6 +184,8 @@ declare global {
   type IEvents = { [event: string]: string }
 
   type ILookup = { [key: string]: any }
+
+  type IPreferences = ILookup
```

Comment:
Should not use `I` preface on interfaces. This will fail eslint rules. Apparently the new recommendation for  TS is to not have the `I` prefix. I will eventually remove from the remote.it.js lib.

Reply:
Hmmm... Ok - is there a new prefix recommendation? Would make it harder to not have prefix at all to name.